src/default-modules/ui/controls/time/time.ts

```diff
@@ -109,11 +123,6 @@ export default class TimeControl {
   }
 
   _updateCurrentTime() {
-    if (this._engine.isDynamicContent) {
-      // TODO: is it same for live ended?
-      this.view.setCurrentTimeBackward(!this._engine.isSyncWithLive);
```

Comment:
Do we still need `setCurrentTimeBackward` logic on `timeView` & in `time.reset()` method?